### PR TITLE
Skip some plugins for install-all

### PR DESCRIPTION
**Pull request by suyograo**
Skip the `*-examples` and `*-jms` plugins for now

### CODE REVIEW

File: tools/Gemfile.plugins.all [hunk 1]
```diff
@@ -1,6 +1,5 @@
 require 'octokit'
-skiplist = Regexp.union([ /^logstash-codec-cef$/, /^logstash-input-gemfire$/, /^logstash-output-gemfire$/,
-    /^logstash-input-couchdb_changes$/, /^logstash-filter-metricize$/, /^logstash-filter-yaml$/, /example/])
+skiplist = Regexp.union([ /^logstash-codec-cef$/, /^logstash-input-gemfire$/, /^logstash-output-gemfire$/, /^logstash-filter-metricize$/, /^logstash-filter-yaml$/, /jms$/, /example$/])
```
Comment (talevy) [inline tools/Gemfile.plugins.all]: not skipping `couchdb_changes` anymore?

Comment (suyograo) [inline tools/Gemfile.plugins.all]: nope, previously couchdb was not published, but we recently did. also, couch is now shipped by default

Comment (untergeek) [inline tools/Gemfile.plugins.all]: w00t!

Comment (talevy) [inline tools/Gemfile.plugins.all]: coool

**Comment by talevy** [comment]
LGTM

**Pull request closed without merging**